tests/unit/components/Player/ItemModal.spec.js

```diff
@@ -4,249 +4,389 @@ import ItemModal from "@/components/Player/ItemModal";
-    expect(wrapper.emitted()).toHaveProperty("toggle-minimize");
-  });
+    it("skips question", () => {
+      const skipQuestionMock = jest.spyOn(ItemModal.methods, "skipQuestion");
+      remount();
+      wrapper
+        .find('[data-test="header"]')
+        .find('[data-test="skip"]')
+        .trigger("click");
 
-  it("skips question", () => {
-    const skipQuestionMock = jest.spyOn(ItemModal.methods, "skipQuestion");
-    const wrapper = mount(ItemModal, {
-      props: {
-        itemList: clonedeep(global.dummyItems),
-        itemDetailList: clonedeep(global.dummyItemDetails),
-      },
-    });
-    wrapper
-      .find('[data-test="header"]')
-      .find('[data-test="skip"]')
-      .trigger("click");
-
-    expect(skipQuestionMock).toHaveBeenCalled();
-    expect(wrapper.emitted()).toHaveProperty("skip-question");
-  });
+      expect(skipQuestionMock).toHaveBeenCalled();
+      expect(wrapper.emitted()).toHaveProperty("skip-question");
+    });
+
+    it("revises question", () => {
+      const reviseQuestionMock = jest.spyOn(ItemModal.methods, "emitRevise");
+      remount();
+      wrapper
+        .find('[data-test="footer"]')
+        .find('[data-test="reviseButton"]')
+        .trigger("click");
 
-  it("revises question", () => {
-    const reviseQuestionMock = jest.spyOn(ItemModal.methods, "emitRevise");
-    const wrapper = mount(ItemModal, {
-      props: {
-        itemList: clonedeep(global.dummyItems),
-        itemDetailList: clonedeep(global.dummyItemDetails),
-      },
-    });
-    wrapper
-      .find('[data-test="footer"]')
-      .find('[data-test="reviseButton"]')
-      .trigger("click");
-
-    expect(reviseQuestionMock).toHaveBeenCalled();
-    expect(wrapper.emitted()).toHaveProperty("revise-question");
+      expect(reviseQuestionMock).toHaveBeenCalled();
+      expect(wrapper.emitted()).toHaveProperty("revise-question");
+    });
   });
 
-  it("submits subjective question", async () => {
-    let responseList = [];
-    global.dummyItems.forEach(() => {
-      responseList.push({
-        answer: null,
+  describe("subjective questions", () => {
+    const itemIndex = 2;
+    const mountWrapper = (responseList = null) => {
+      if (responseList == null) {
+        responseList = [];
+        global.dummyItems.forEach(() => {
+          responseList.push({
+            answer: null,
+          });
+        });
+      }
+
+      wrapper = mount(ItemModal, {
+        props: {
+          itemList: clonedeep(global.dummyItems),
+          itemDetailList: clonedeep(global.dummyItemDetails),
+          responseList: responseList,
+          selectedItemIndex: itemIndex,
+        },
       });
+    };
+
+    it("submits question", async () => {
+      let draftResponses = clonedeep(global.dummyItemResponses);
+      const submitQuestion = jest.spyOn(ItemModal.methods, "submitQuestion");
+
+      mountWrapper();
+      // enter some value in the input field
```

Comment:
In this describe, you're not mounting at `beforeEach` but rather mounting at each individual tests, but in the test suite `describe("checkbox questions", () => {`, you have a `beforeEach` and you're mounting in that.
So, any particular reason for NOT beforeEach mounting in `subjective question` describe and mounting beforeEach in `checkbox question` describe?